File: Latihan2/Hardware.java
package Latihan2;

class Hardware extends Product{
    private String brand;
    private String model;

    // Constructor
    public Hardware(){
    }

    // Get-set Brand
    public String getBrand(){
        return brand;
    }

    public void setBrand(String brand){
        this.brand = brand;
    }

    // Get-set Model
    public String getModel(){
        return model;
    }

    public void setModel(String model){
        this.model = model;
    }
}

File: Latihan2/Hardware.php
<?php

class Hardware extends Product{
    private $brand;
    private $model;

    public function __construct(){
    }

    // Get-set Brand
    public function setBrand($brand){
        $this->brand = $brand;
    }

    public function getBrand(){
        return $this->brand;
    }

    // Get-set Model
    public function setModel($model){
        $this->model = $model;
    }

    public function getModel(){
        return $this->model;
    }
}

?>